release notes/v0.43.0.md

```diff
@@ -0,0 +1,124 @@
+k6 v0.43.0 is here! :tada:
+
+Notable changes in this release include:
+- [xk6-browser](https://github.com/grafana/xk6-browser) is now bundled in k6 as an experimental module, and usable without a separate binary or compilation step!
+- Native support for JavaScript's `async`/`await`.
+- A new experimental JavaScript module for distributed tracing.
+- Large refactoring of core components to simplify the code base, improve maintainability, and increase test coverage.
+- Bug fixes, UX improvements, and maintenance.
+
+Keep reading for the details.
+
+## Breaking changes
+
+- [#2807](https://github.com/grafana/k6/pull/2807) Use non-zero exit codes for tests aborted by Ctrl+C or the REST API.
+
+  Aborting a test run with Ctrl+C will now exit with code `105`, and stopping via the REST API will exit with code `103`.
+
+- [#2903](https://github.com/grafana/k6/pull/2903) JS modules are now cached by absolute, and not relative paths.
+
+  This fixes a regression introduced in v0.42.0 ([#2773](https://github.com/grafana/k6/pull/2773)), but has the side-effect of changing how modules are cached internally, which may cause issues for scripts that relied on the previous behavior.
+
+
+## New Features
+
+### xk6-browser is now a built-in module [#2884](https://github.com/grafana/k6/pull/2884)
+
+[xk6-browser](https://github.com/grafana/xk6-browser), a k6 extension that enables browser automation and end-to-end testing from k6 scripts, now comes bundled with k6 under the `k6/experimental/browser` import path. This makes it more accessible, as users no longer need to build their own binary with [xk6](https://github.com/grafana/xk6), or use a separate binary to run browser tests.
+
+All current xk6-browser scripts that work with v0.8.0 will continue to work with the built-in module in k6 v0.43.0, by simply changing the import path from `k6/x/browser` to `k6/experimental/browser`, and setting the environment variable `K6_BROWSER_ENABLED` to `true`. You can see a script example [here](https://github.com/grafana/k6/blob/v0.43.0/samples/experimental/browser.js), and read the updated [xk6-browser documentation](https://k6.io/docs/javascript-api/xk6-browser/).
```

Comment:
Should we expand here on why the environment variable is needed?

Reply:
That might be a good idea, but can we also add a line that it is *not* supported in the cloud by default

Reply:
Since xk6-browser is no longer an extension, we've been updating the docs and renaming the name to "browser module". The new path would be "the [browser module documentation](https://k6.io/docs/javascript-api/k6-browser/)".

Reply:
Alright. Let me know if 2d716737 and 34e5d70e resolve this.